manim/mobject/types/vectorized_mobject.py

```diff
@@ -2022,6 +2022,43 @@ def __init__(self, vmobject, **kwargs):
 
 
 class DashedVMobject(VMobject, metaclass=ConvertToOpenGL):
+    """
+
+    Examples
+    --------
+    .. manim:: DashedVMobjectExample
+        :save_last_frame:
+
+        class DashedVMobjectExample(Scene):
+            def construct(self):
+                r = 0.5
+                pos = 2.5 * UP + 6 * LEFT  # Tracker for position on screen
```

Comment:
I'd prefer using `VGroup()` and `.arrange` instead of the `pos` approach, it's simpler and the type of code we should want to encourage